app/about_us/our_partners/page.js

```diff
@@ -0,0 +1,92 @@
+import styles from "./page.module.css";
+
+const OurPartners = () => {
+  // const partnerImages = [
+  //   "alphabet-logo.png",
+  //   "amazon_logo.png",
+  //   "CBC_Logo_White.png",
+  //   "Microsoft-Logo-white.png",
+  //   "nyu-logo.png",
+  //   "QueensLogo_white.png",
+  //   "samsung-logo.png",
+  //   "sodexo-logo.png",
+  // ];
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.ourPartner}>
+        <p>Our Partners</p>
+        <p>
+          We collaborate with some of the most respected names in the space and
+          technology industries to make every journey extraordinary.
+        </p>
+      </div>
+      <section className={styles.partner}>
+        {/* {partnerImages.map((fileName, index) => {
+          console.log(`${fileName}`);
+          console.log(`${index}`);
+          <img
```

Comment:
This one I've showed you after class 😄 
```
{partnerImages.map((fileName, index) => (
          <img 
            ....
          />
 )}
```

Reply:
The diff is the type of brackets!

Reply:
Yes noted and now it is working fine. Thanks for your extra time